src/clients/composite/aws_route53_client.py

```diff
@@ -91,8 +95,27 @@ def _create_query_log_actions(
         queryLogActionList.append(CreateQueryLogAction(account, self._route53, self._iam, self._config, hostedZone.id))
         return queryLogActionList
 
-    def _route53_log_group_enforcement_actions(self, with_subscription_filter: bool) -> Sequence[ComplianceAction]:
+    def _route53_query_logs_resource_policy_document(self, account: Account) -> str:
```

Comment:
This feels a bit weird being hard coded like this here, want to talk me through it?